consultation_analyser/consultations/jinja2/consultations/questions/show.html

```diff
@@ -93,46 +93,61 @@ <h2 class="govuk-!-margin-bottom-2 govuk-!-margin-top-0">Prevalent themes</h2>
-                  {% if theme.answer_count and highest_theme_count %}
-                    <span class="iai-bar__bar js-bar" style="width: {{ (theme.answer_count / highest_theme_count) * 100 }}%;"></span>
-                  {% endif %}
-                </bar-animation>
-              </td>
-            </tr>
+            {% if not theme.is_outlier %}
+              <tr class="govuk-table__row">
+                <td class="govuk-table__cell">
+                  <details class="govuk-details govuk-!-margin-bottom-0">
+                    <summary class="govuk-details__summary govuk-!-margin-top-1">
+                      <span class="govuk-details__summary-text">{{ theme.short_description }}</span>
+                    </summary>
+                    <div class="govuk-details__text">
+                      <p class="govuk-!-margin-bottom-0">{{ theme.summary }}</p>
+                      <br>
+                      <p class="govuk-!-margin-bottom-0">This theme has the following keywords:</p>
+                      <ul class="iai-inline-list govuk-!-margin-top-1">
+                        {% for word in theme.topic_keywords %}
+                          <li class="iai-inline-list__item">{{ word }}</li>
+                        {% endfor %}
+                      </ul>
+                    </div>
+                    <p class="govuk-body"><a class="govuk-link" href="{{ url("question_responses", kwargs={"consultation_slug":
+                        consultation_slug, "question_slug" : question.slug, "section_slug" : question.section.slug},
+                        query_kwargs={"theme": theme.id}) }}">View responses</a></p>
+                  </details>
+                </td>
+                <td class="govuk-table__cell">
+                  <bar-animation class="iai-bar">
+                    <span class="iai-bar__value">{{ theme.answer_count }}</span>
+                    {% if theme.answer_count and highest_theme_count %}
+                      <span class="iai-bar__bar js-bar" style="width: {{ (theme.answer_count / highest_theme_count) * 100 }}%;"></span>
+                    {% endif %}
+                  </bar-animation>
+                </td>
+              </tr>
+            {% endif %}
           {% endfor %}
         </tbody>
       </table>
       {% if blank_free_text_count and question.has_free_text %}
         {% if blank_free_text_count == 1 %}
           <p class="govuk-body">There is {{ blank_free_text_count }} response with no free text and this has been excluded from the theme analysis.</p>
         {% else %}
-          <p class="govuk-body">There were {{ blank_free_text_count }} responses with no free text, and these have been excluded from the theme analysis.</p>
+          <p class="govuk-body">There are {{ blank_free_text_count }} responses with no free text, and these have been excluded from the theme analysis.</p>
         {% endif %}
       {% endif %}
+
+      {% if outliers_count %}
+        {% if outliers_count == 1 %}
```

Comment:
Nice. I think we could use a `jinja` `with` statement to clean this up?

```
{% with link=url("question_responses", kwargs={"consultation_slug":
            consultation_slug, "question_slug" : question.slug, "section_slug" : question.section.slug},
            query_kwargs={"theme": outlier_theme_id}) %}
  {% if outliers_count == 1 %}
    <p class="govuk-body">There is <a class="govuk-link" href="{{link}}">1 response which is an outlier</a> and does not fit into any of the existing themes.</p>
  {% else %}
        <p class="govuk-body">There are <a class="govuk-link" href="{{link}}">{{ outliers_count }} responses which are outliers</a> and do not fit into any of the existing themes.</p>
    # etc
  {% endif %}
{% endwith %}

Reply:
Yes, that looks a lot nicer! Will update.